### DISCUSSION TITLE
Unable to show multiple tile layers

### DISCUSSION THREAD

**Discussion by cubocicloide** [visgl/deck.gl#6676] [Q&A] [answered]
I am evaluating your awesome library, but unfortunately I am stuck with the following.
I am trying to show multiple layers but, I do not know why, only the first is showing.
[This codesandbox](https://codesandbox.io/s/vn7qwp) contains what I am trying to achieve. In particular, in App.js, I am defining two TileLayer:

- tileLayerAfrica
- tileLayerNASA

Then, I am including both in the layers prop of DeckGL (as `layers={[tileLayerAfrica, tileLayerNASA]}`).
The problem is that only tileLayerAfrica is showing up. How can I show them both?

PS: if I set `layers={[tileLayerNASA, tileLayerAfrica]}`, only tileLayerNASA will be shown.

**Comment by Pessimistress** [comment, marked as answer]
I cannot tell you what's wrong without seeing your code. I'm guessing you are not assigning unique ids to each layer. A good place to start is https://deck.gl/docs/developer-guide/using-layers

**Reply by cubocicloide** [reply]
@Pessimistress , thank you, it worked!
